zerver/models/recipients.py

```diff
@@ -112,44 +114,49 @@ def bulk_get_direct_message_group_user_ids(recipient_ids: List[int]) -> Dict[int
     return result_dict
 
 
-class Huddle(models.Model):
+class DirectMessageGroup(models.Model):
     """
     Represents a group of individuals who may have a
     group direct message conversation together.
 
-    The membership of the Huddle is stored in the Subscription table just like with
-    Streams - for each user in the Huddle, there is a Subscription object
-    tied to the UserProfile and the Huddle's recipient object.
+    The membership of the DirectMessageGroup is stored in the Subscription
+    table just like with Streams - for each user in the DirectMessageGroup,
+    there is a Subscription object tied to the UserProfile and the
+    DirectMessageGroup's recipient object.
 
     A hash of the list of user IDs is stored in the huddle_hash field
     below, to support efficiently mapping from a set of users to the
-    corresponding Huddle object.
+    corresponding DirectMessageGroup object.
     """
 
     # TODO: We should consider whether using
     # CommaSeparatedIntegerField would be better.
     huddle_hash = models.CharField(max_length=40, db_index=True, unique=True)
-    # Foreign key to the Recipient object for this Huddle.
+    # Foreign key to the Recipient object for this DirectMessageGroup.
     recipient = models.ForeignKey(Recipient, null=True, on_delete=models.SET_NULL)
 
+    class Meta:
+        db_table = "zerver_huddle"
```

Comment:
Let's add a comment explaining that this is the old name and should change.

Reply:
Added 
```
# TODO: As a part of migration of Huddle to DirectMessageGroup, the
# "zerver_huddle" table needs to be renamed to "zerver_directmessagegroup".
```